app/ide-desktop/lib/client/src/authentication.ts

```diff
@@ -156,38 +156,49 @@ export function onOpenUrl(url: URL, window: () => electron.BrowserWindow) {
  * The credentials file is placed in the user's home directory in the `.enso` subdirectory
  * in the `credentials` file. */
 function initSaveAccessTokenListener() {
-    electron.ipcMain.on(ipc.Channel.saveAccessToken, (event, accessToken: string | null) => {
-        event.preventDefault()
+    electron.ipcMain.on(
+        ipc.Channel.saveAccessToken,
+        (event, accessTokenPayload: SaveAccessTokenPayload | null) => {
+            event.preventDefault()
```

Comment:
this event has a default behavior?! :o

Reply:
https://github.com/enso-org/enso/pull/8278/files#diff-af05dad38b444af841a7c3095aa4540d7871056a32149c7c373d66fc94c08dd9R162

Seems like :)